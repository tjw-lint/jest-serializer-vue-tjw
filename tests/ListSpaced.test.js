const helpers = require('./helpers.js');

import { shallowMount } from '@vue/test-utils';
import ListSpaced from './components/ListSpaced.vue';

describe('ListSpaced.vue', () => {
  test('Snapshot unchanged', () => {
    helpers.mockSettings({});
    const wrapper = shallowMount(ListSpaced);

    expect(wrapper)
      .toMatchSnapshot();
  });

  test('Default pretty options', () => {
    helpers.mockSettings({
      pretty: true
    });
    const wrapper = shallowMount(ListSpaced);

    expect(wrapper)
      .toMatchSnapshot();
  });
});
