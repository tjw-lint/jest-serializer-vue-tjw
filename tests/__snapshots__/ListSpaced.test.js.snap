// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ListSpaced.vue Default pretty options 1`] = `
<aside role="banner" class="test"><a title="one" class="one">
    one
  </a> <a title="two" class="two">
    two
  </a> <a title="three" class="three">
    three
  </a> <a title="four" class="four">
    four
  </a></aside>
`;

exports[`ListSpaced.vue Snapshot unchanged 1`] = `
<aside role="banner" class="test">
  <a title="one" class="one">
    one
  </a>
  <a title="two" class="two">
    two
  </a>
  <a title="three" class="three">
    three
  </a>
  <a title="four" class="four">
    four
  </a>
</aside>
`;
